samples/dnn/models.yml

```diff
@@ -178,3 +178,16 @@ fcn8s:
   height: 500
   rgb: false
   sample: "segmentation"
+
+fcnresnet101:
+  load_info:
+    url: "https://github.com/richard28039/fcnresnet101_onnx_model/raw/main/fcn_resnet101.onnx?download="
+    sha1: "1242104e22c9eb0b320d7ae36b11cdd1fda40857"
+  model: "fcn_resnet101.onnx"
+  mean: [103.5, 116.2, 123.6]
+  scale: 0.019
+  width: 500
+  height: 500
+  rgb: false
+  classes: "object_detection_classes_pascal_voc.txt"
```

Comment:
Please add an image with resulting segmentation from a sample launch. I have a concern that classes might be 1 position shifted (depends on background id).

So please run a sample on an image with more that one segmented object and put to PR description or in this thread,

Reply:
Hello @dkurt , I think your concern is right, the background classes might have to be add.
Should I add background class to https://github.com/opencv/opencv/blob/4.x/samples/data/dnn/object_detection_classes_pascal_voc.txt ?

without background
![image](https://github.com/opencv/opencv/assets/89371302/17efac9a-eb57-421f-8d99-61b2678335a7)
![image](https://github.com/opencv/opencv/assets/89371302/9a503184-9ef8-4838-b6cd-dcdd937bd43a)

with background
![image](https://github.com/opencv/opencv/assets/89371302/a3632579-0c32-4855-aa3a-929d9f3d5cd1)
![image](https://github.com/opencv/opencv/assets/89371302/caca0d81-533e-471a-87e3-6e43528ad3f1)

Reply:
Thanks, problem should be solved in a common.py script which will affect other samples so I'd like to merge this PR as is.

Reply:
I got it, thank you!